specs/protocol/delta/span-batches.md

```diff
@@ -341,6 +341,8 @@ Span-batch rules, in validation order:
       It may reach to the next origin outside the L1 origins of the span.
   - Rules:
     - For each `block_input` whose timestamp is greater than `safe_head.timestamp`:
+      - `block_input.l1_origin.number < safe_head.l1_origin.number` -> `drop`: enforce increasing L1
+        origins (also for non-overlapping section of overlapping span batches).
```

Comment:
The part in parentheses I think is going to be hard to understand when returning to this without context. Can you say what the span batch is overlapping with? That might help.